src/connectedk8s/azext_connectedk8s/_precheckutils.py

```diff
@@ -154,19 +150,22 @@ def executing_cluster_diagnostic_checks_job(corev1_api_instance, batchv1_api_ins
                         exception_occured_counter = 1
                 # If any exception occured we will print the exception and return
                 if exception_occured_counter == 1:
-                    logger.warning("Cleanup of previous diagnostic checks helm release failed and hence couldn't \
-                        install the new helm release. Please cleanup older release using \"helm delete \
-                            cluster-diagnostic-checks -n azure-arc-release\" and try onboarding again")
+                    logger.warning("Cleanup of previous diagnostic checks helm release failed and hence couldn't "
+                                   "install the new helm release. Please cleanup older release using \"helm delete "
+                                   "cluster-diagnostic-checks -n azure-arc-release\" and try onboarding again")
                     telemetry.set_exception(exception=error_kubectl_delete_helm.decode("ascii"),
                                             fault_type=consts.Cluster_Diagnostic_Checks_Release_Cleanup_Failed,
-                                            summary="Error while executing cluster diagnostic checks Job")
+                                            summary="Error while executing Cluster Diagnostic Checks Job")
                     return
 
         chart_path = azext_utils.get_chart_path(consts.Cluster_Diagnostic_Checks_Job_Registry_Path, kube_config,
                                                 kube_context, helm_client_location,
                                                 consts.Pre_Onboarding_Helm_Charts_Folder_Name,
                                                 consts.Pre_Onboarding_Helm_Charts_Release_Name, False)
 
+        print("Step: {}: Chart path for Cluster Diagnostic Checks Job: {}".format(azext_utils.get_utctimestring(),
+                                                                                  chart_path))
+        print("Step: {}: Creating Cluster Diagnostic Checks job".format(azext_utils.get_utctimestring()))
```

Comment:
It is better to use logger instead of print function.

Reply:
These messages are not Warnings, so didn't want to use logger.Warning.  I want these to show up even if the user runs it without --debug.  For some reason logger.Info isn't working - it doesn't output to stdout, which is why I am using print.